LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDeviceTelemetry.cs

```diff
@@ -71,6 +71,9 @@ public class LoRaDeviceTelemetry
         [JsonProperty("edgets")]
         public long Edgets { get; set; }
 
+        [JsonProperty("dupmsg")]
```

Comment:
Just to see if I understand correctly.
If you don't have an strategy for the device `"dupmsg"` won't be part of the telemetry.

On the other hand, if you have an strategy for the device the `"dupmsg"` will be present, for true or false.

Is that correct?

Reply:
It will only be included if it is true. That's basically "marking" the message. Avoiding more data to be sent to the server than necessary.

Reply:
Ok. In case you don't know, the same could be achieved using a DefaultValueHandling on the property: 

```c#
[JsonProperty("dupmsg", DefaultValueHandling = DefaultValueHandling.Ignore)]
public bool IsDuplicated { get; set; }
```

Sample: https://dotnetfiddle.net/dUoPJe